cumulus/parachains/runtimes/glutton/glutton-kusama/src/lib.rs

```diff
@@ -204,12 +232,18 @@ construct_runtime! {
 			Pallet, Call, Config<T>, Storage, Inherent, Event<T>, ValidateUnsigned,
 		} = 1,
 		ParachainInfo: parachain_info::{Pallet, Storage, Config<T>} = 2,
+		// TODO: probably not needed now, but I'm not sure
+		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 3,
 
 		// DMP handler.
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin} = 10,
 
+		// Collator support
+		Aura: pallet_aura::{Pallet, Storage, Config<T>} = 23,
+		AuraExt: cumulus_pallet_aura_ext::{Pallet, Storage, Config<T>} = 24,
+
 		// The main stage.
-		Glutton: pallet_glutton::{Pallet, Call, Storage, Event, Config<T>} = 20,
+		Glutton: pallet_glutton::{Pallet, Call, Storage, Event, Config<T>} = 30,
```

Comment:
Changing indices is generally a no-no. Probably fine for Glutton since we don't have any in the wild. But what is the reason for this change?

Reply:
I was shuffling them around while building the runtime and I didn't check the original indices when I posted the PR. Good catch, restored it now.